pacemaker.spec.in

```diff
@@ -1,77 +1,113 @@
-%global gname haclient
+# Define globals to control package behavior (configure these as desired)
+
+## User and group to use for nonprivileged services
 %global uname hacluster
+%global gname haclient
+
+## Where to install Pacemaker documentation
 %global pcmk_docdir %{_docdir}/%{name}
 
-%global specversion 1
+## Upstream pacemaker version, and its package version 
 %global pcmkversion 1.1.15
-# set following to the actual commit or, for final release, concatenate
-# "pcmkversion" macro to "Pacemaker-" (will yield a tag per the convention)
+%global specversion 1
+
+## Upstream commit (or git tag, such as "Pacemaker-${pcmkversion}"
+## for an official release) to use for this package
 %global commit HEAD
+
+
+# Define globals for convenient use later
+
+## Workaround to use parentheses in other globals
 %global lparen (
 %global rparen )
+
+## Short version of git commit
 %global shortcommit %(c=%{commit}; case ${c} in
                       Pacemaker-*%{rparen} echo ${c:10};;
                       *%{rparen} echo ${c:0:7};; esac)
+
+## Whether this is a release candidate
 %global pre_release %(s=%{shortcommit}; [ ${s: -4:3} != -rc ]; echo $?)
+
+## Whether this is a development branch
 %global post_release %([ %{commit} = Pacemaker-%{shortcommit} ]; echo $?)
+
+## Github organization that distributes source
 %global github_owner ClusterLabs
 
-# Turn off the auto compilation of python files not in the site-packages directory
-# Needed so that the -devel package is multilib compliant
+## Turn off auto-compilation of python files outside site-packages directory,
+## so that the -devel package is multilib-compliant
 %global __os_install_post %(echo '%{__os_install_post}' | sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g')
 
+## Heuristic used to infer brokenness of some tools
```

Comment:
Or is it meant for the list growing in the future?  We should rename the variable then.